lib/plugins/aws/logs.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-const BbPromise = require('bluebird');
 const _ = require('lodash');
 const dayjs = require('dayjs');
 const utc = require('dayjs/plugin/utc');
 const validate = require('./lib/validate');
 const formatLambdaLogEvent = require('./utils/formatLambdaLogEvent');
+const timers = require('../../utils/timers');
```

Comment:
Let's use `timers-ext/sleep` instead

Reply:
I missed that.

I tried to change it but it's not trivial since it's stubbed by the tests and I'm not sure how to stub function that's directly required as main export from a module (if possible).

Also, considering that the implementation of such timer is
```js
async function sleep(interval = 0) {
  await new Promise(resolve => setTimeout(resolve, interval));
}
```
Do you think that's worth having a dependency just for that or could we remove the dep in a separate PR?

Reply:
>  I'm not sure how to stub function that's directly required as main export from a module (if possible).

We address such cases usually with `proxyquire`

Reply:
> Do you think that's worth having a dependency

yes, we prefer to rely on single toolkit served as external package (which is well backed by tests), instead of copying those utils in every package we find use for it